WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -1,17 +1,43 @@
 <% provide(:title, 'Competitions') %>
 
 <div class="container">
-  <h1><%= yield(:title) %></h1>
+  <h2><%= yield(:title) %></h2>
 
-  <%= link_to "New competition", new_competition_path %>
-  <table class="table">
-    <% @competitions.each do |competition| %>
+  <%= form_tag competitions_path(anchor: "today"), method: :get, class: "competition-select" do %>
+    <table>
```

Comment:
So I see here you're using a table, whereas on the WCA website we have a bunch of div.form-group instead. Was this departurn intentional? I feel like the table isn't going to look at nice on mobile. Check out how the current WCA website looks on a small screen:

![2016-01-15_717x460_kaladin](https://cloud.githubusercontent.com/assets/277474/12365944/dac95fcc-bb8c-11e5-92b0-c07e05e1e3e1.png)